.gitignore

```diff
@@ -328,3 +328,4 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+src/Arcus.Messaging.Tests.Integration/appsettings.local.json
```

Comment:
nitpick. may want to do this for all local appsettings? So no fixed paths are here?